docs/hugo/content/guide/asov1-asov2-migration/_index.md

```diff
@@ -0,0 +1,331 @@
+---
+title: ASOv1 to ASOv2 migration guide
+weight: 5
+layout: single
+cascade:
+  - type: docs
+  - render: always
+---
+ASOv1 is [deprecated](https://github.com/Azure/azure-service-operator/blob/main/README.md#aso-v1). 
+We strongly recommend migrating to ASOv2 to continue getting new features and support.
+
+## Assumptions
+
+- Migrating a single cluster.
+  - If you have ASOv1 installed on multiple clusters, apply these steps to each cluster.
+- ASOv1 is installed in its default namespace `azureoperator-system`.
+  - If it's installed into a different namespace, replace all occurrences of `azureoperator-system` in this document with 
+    the namespace it's installed into. 
+- ASOv2 is not yet installed in the cluster.
+- ASOv1 is managing a single subscription, so ASOv2 will also be managing that same single subscription via a single 
+  global credential.
+  - See [ASOv2 credential scope](../authentication/#credential-scope) for other credential scopes that allow for 
+    managing multiple subscriptions and/or using multiple identities.
+
+## Performing the migration
+
+### Check the version of cert-manager
+
+Ensure the version of cert-manager in your cluster supports `v1` resources.
+
+```
+kubectl api-resources --api-group cert-manager.io`
+```
+should show `v1` resources.
+
+> ℹ️ **Note:** We strongly recommend ensuring that you're running the latest version of cert-manager 
+> (1.14.4 at the time this article was written).
+
+> ℹ️ **Note:** We also recommend ensuring that ASOv1 is configured to use `cert-manager.io/v1` resources.
+> You can run `helm upgrade` and pass `--set certManagerResourcesAPIVersion=cert-manager.io/v1` to ensure ASOv1 is using
+> the v1 versions of the cert-manager resources.
+
+### Install ASOv2
+
+Follow the [standard instructions](../../#installation). We recommend you use the same credentials as ASOv1 is currently using.
+
+> ℹ️ **Note:** Make sure to follow the [guidelines](../crd-management/) for setting the `--crdPattern`. Configure ASOv2 
+> to only manage the CRDs which you need.
+> 
+> For example: if you're using storage accounts, resource groups, redis cache's, cosmosdbs, eventhubs, and SQL Azure, 
+> then you'd configure 
+> `--set crdPattern='resources.azure.com/*;cache.azure.com/*;documentdb.azure.com/*;eventhub.azure.com/*;sql.azure.com/*;storage.azure.com/*'`
+
+### Stop ASOv1 reconciliation
+
+Mark each ASOv1 resource with the `skipreconcile=true` annotation.
```

Comment:
This must be hard to put annotation on _each_ resource. 
Should have a way in ASOv2 to pause reconciliation at the controller level for all resources, if not already there?

Reply:
I give commands just below of how to do it for a whole namespace at once. It's a one-liner so I don't think we need any other features. At most, we could make an `asoctl` feature to do the one-line like `asoctl annotate all` or something, but I don't think it's needed atm really.